packages/api/src/controllers/room.ts

```diff
@@ -17,18 +21,119 @@ app.post("/", authorizer({}), async (req, res) => {
     req.config.livekitApiKey,
     req.config.livekitSecret
   );
-  const opts = {
+  const room = await svc.createRoom({
     name: id,
     // timeout in seconds
-    emptyTimeout: 30 * 60,
-    maxParticipants: 2,
-  };
-  const room = await svc.createRoom(opts);
+    emptyTimeout: 15 * 60,
+    maxParticipants: 10,
+    metadata: `{"userId": "${req.user.id}"}`,
```

Comment:
Nice to have the metadata! As a nit, you could write a syntax-safe JSON here as well:
```suggestion
    metadata: JSON.stringify({userId: req.user.id}),
```